infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -59,7 +59,9 @@
 
     public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, String localClusterId) {
         this.runtime = rt;
-        this.corfuStore = new CorfuStore(runtime);
+        // LR does not require transaction logging as we don't want data change notifications on the replicated data
```

Comment:
is the relation between data change notification and tx stream general to corfu?

Reply:
not really, it's one use case (out of Corfu) but it can be used for any purpose. Just didn't know how to make it clear.. I've rephrased it, take a look and let me know if you think it still is confusing.

Reply:
maybe we can say - 'Replicated entries from LR need not be logged to the transaction stream'... or something like that?